modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/CommentResourceImpl.java

```diff
@@ -191,6 +252,94 @@ public EntityModel getEntityModel(MultivaluedMap multivaluedMap) {
 		return new CommentEntityModel();
 	}
 
+	@Override
+	public Comment
+			getSiteBlogPostingByExternalReferenceCodeBlogPostingExternalReferenceCodeCommentByExternalReferenceCode(
+				Long siteId, String blogPostingExternalReferenceCode,
+				String externalReferenceCode)
+		throws Exception {
+
+		BlogsEntry blogsEntry = _getBlogsEntry(
```

Comment:
Very nice work making that each use case follows the same structure in the code: _getEntity, _getComment, checkpermission, return comment. Same with the other use cases 👏